src/applications/static-pages/facilities/BasicFacilityListWidget.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { apiRequest } from '../../../platform/utilities/api';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+import { buildAddressArray } from '../../facility-locator/utils/facilityAddress';
+
+export default class BasicFacilityListWidget extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: true,
+    };
+  }
+
+  componentDidMount() {
+    const facilityIds = Object.keys(this.props.facilities);
+    this.request = apiRequest(
+      `/facilities/va?ids=${facilityIds}`,
```

Comment:
Is there anything different about how the facility ids are treated on this page? I'm not seeing the list show up locally.

Reply:
They should be the same. 

They show on my local with:
BASE_URL: 'https://dev.va.gov',
API_URL: 'https://dev-api.va.gov',
pulling data from http://stg.va.agile6.com.

Reply:
It works on my local (getting data from http://stg.va.agile6.com and using BASE_URL: 'https://dev.va.gov', API_URL: 'https://dev-api.va.gov').